<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
          <item>
           <widget class="Map" name="map" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>611</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lapLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="splitLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="plotScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="comPortPushButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Select COM Port</string>
        </property>
       </widget>
       <widget class="QPushButton" name="replayPushButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>90</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap_Soruce">
    <property name="title">
     <string>Map Source</string>
    </property>
    <addaction name="actionMapBox"/>
    <addaction name="actionGoogle"/>
   </widget>
   <addaction name="menuMap_Soruce"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMapBox">
   <property name="text">
    <string>MapBox</string>
   </property>
  </action>
  <action name="actionGoogle">
   <property name="text">
    <string>Google</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Map</class>
   <extends>QWidget</extends>
   <header>map.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
